### ISSUE TITLE
Only error color used

### ISSUE THREAD

**Issue by fibric** [oderwat/vscode-indent-rainbow#12]
Hi,

since very recently I see only error color being used instead of the configured colors.
![screen shot 2016-09-24 at 12 21 49 pm](https://cloud.githubusercontent.com/assets/623932/18807817/7b7b7ac8-8251-11e6-8d82-1d769120a698.png)

This applies to both, VScode (Version 1.5.3) and VScode (Version 1.6.0-insider).

``` javascript
{
...
// Indent-Rainbow configuration

  // For which languages indent-rainbow should be activated (if empty it means all).
  "indentRainbow.includedLanguages": [],

  // For which languages indent-rainbow should be deactivated (if empty it means none).
  "indentRainbow.excludedLanguages": [],

  // The delay in ms until the editor gets updated.
  "indentRainbow.updateDelay": 100,

  // Automatically change indent settings for languages (see README.md for details).
  "indentRainbow.indentSetter": {},

  // An array with rgba strings which are used as colors.
  "indentRainbow.colors": [
    "rgba(64,64,16,0.3)",
    "rgba(32,64,32,0.3)",
    "rgba(64,32,64,0.3)",
    "rgba(16,48,48,0.3)",
    "rgba(128,32,32,0.3)"
  ],
...
}
```

**Comment by oderwat** [comment]
I use 1.5.3 personally and it seems to still work for me. Make sure you don't have indent using tabs active in this editor (click on the "Spaces 4" in the bottom status line of the editor). You could also let VSCode detect the correct settings there. I guess there is something not communicating right into what the plugin thinks the settings are.

**Comment by fibric** [comment]
Thanks, but it's 4 Spaces as you can see on the picture above.

**Comment by oderwat** [comment]
Yes I saw that. But did you try to toggle it, restart and toggle it back? Change it to 2 Spaces.. I can't reproduce. There may be other effects I don't see.

**Comment by oderwat** [comment]
Probably another plugin you added recently changed something such that there is now some conflict with your configuration. Did you try to remove the other plugins?

**Comment by fibric** [comment]
Okay, I think it's the [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) extension which causes the issue.

I uninstalled everything, then started to reinstall my required extensions one by one starting with indent-rainbow.

When I installed, the mentioned EditorConfig extension the indentation color changed to red.

**Comment by oderwat** [comment]
I remember that I tried this plugin too and that I had problems getting it running too. That was one of the reasons why I added the experimental automatic config in the plugin. Which worked but I didn't want to support it. After VSCode itself now has decent detection I just removed all of my configurations and just use the native detection code for indentation. I am not sure what editor config adds what I am missing though.
